src/app/create/[id]/createBlueprintSteps/ExtractFields.tsx

```diff
@@ -837,10 +834,9 @@ const ExtractFields = ({
                   </Button>
                 </div>
                 {/* Only show output if there are regex parts and valid outputs */}
-                {parseRegexParts(regex.parts).length > 0 &&
-                regexGeneratedOutputs[index] !== undefined &&
-                regexGeneratedOutputs[index] !== null &&
-                regexGeneratedOutputs[index].length > 0 ? (
+                {
+                parseRegexParts(regex.parts).length > 0 ||
+                regexGeneratedOutputs.length > 0 ? (
```

Comment:
Here the individual elems can be undefined and null and that will still be valid as we are only checking the length. Why are we doing this?